python/source/state_representation/bind_jacobian.cpp

```diff
@@ -53,7 +54,14 @@ void bind_jacobian(py::module_& m) {
     return jacobian.solve(twist);
   }, "Solve the system dX = J*dq to obtain dq which is more efficient than multiplying with the pseudo-inverse");
 
-  // TODO access operators
+  c.def("__getitem__", [](const Jacobian& jacobian, std::tuple<int, int> coefficients) {
+    return jacobian(std::get<0>(coefficients), std::get<1>(coefficients));
+  }, "Overload the [] operator to modify the value at given (row, col)");
+  c.def("__setitem__", [](Jacobian& jacobian, std::tuple<int, int> coefficients, double value) {
+    jacobian(std::get<0>(coefficients), std::get<1>(coefficients)) = value;
+    return jacobian;
```

Comment:
to properly respect the C++ API, this should return the value at the index, not the jacobian itself
```suggestion
    return value;
```